First version of Kura 3.0.0 release notes.

MMaiero opened the pull request:
Applied formatter to Api pom file.

Signed-off-by: MMaiero <matteo.maiero@eurotech.com>

ctron commented on kura/distrib/RELEASE_NOTES.txt:
```diff
@@ -1,3 +1,251 @@
+  * Added support for MQTT over Websockets
+  * Added support for AWS IoT and Azure IoT platforms
+  * Simple JSON is now a supported encoding option in Eclipse Kura Cloud Service
+  * Added support for Fedora 25 systems
```
I would limit this to "ARM" support, as standard fedora 25 (x86_64) is not supported.

MMaiero committed:
First version of Kura 3.0.0 release notes.

Applied formatter to Api pom file.

Signed-off-by: MMaiero <matteo.maiero@eurotech.com>

MMaiero committed:
Updated release notes.

Added references to issue #1335.
Added references to PRs: #1352, #1350, #1349, #1347 and #1344.

Signed-off-by: MMaiero <matteo.maiero@eurotech.com>
